### PR TITLE
KEYCLOAK-9987 Upgrade to Wildfly 17.0.1.Final

### PR DESCRIPTION
**Pull request by vramik**

### CODE REVIEW

**Review by hmlnarik — DISMISSED** [review]

File: upgrading/topics/keycloak/changes.adoc [hunk 1]
```diff
@@ -1,5 +1,23 @@
 == Migration Changes
 
+=== Migrating to 7.0.0
+
+==== Upgrade to Wildfly 17
+
+The {project_name} server was upgraded to use Wildfly 17 as the underlying container. This does not directly involve any
+specific {project_name} server functionality, but there are few changes related to the migration, which worth mentioning.
```
Comment (andymunro) [inline upgrading/topics/keycloak/changes.adoc]: suggest:  

- there are "a" few changes

- which "are" worth mentioning

**Review by andymunro — CHANGES_REQUESTED** [review]
see the comments above.

**Comment by vramik** [comment]
Thanks @andymunro, incorporated

**Review by hmlnarik — CHANGES_REQUESTED** [review]
Please update topics/templates/document-attributes-product.adoc similarly (feel free to change CD version to 7 along the way)

**Review by andymunro — DISMISSED** [review]

**Review by hmlnarik — DISMISSED** [review]
Thanks @vramik !

File: topics/templates/document-attributes-product.adoc [hunk 2]
```diff
@@ -18,10 +18,10 @@ endif::[]
 
 ifeval::[{project_product_cd}==true]
 :project_name_full: Red Hat Single Sign-On Continuous Delivery
-:project_version: 6
-:project_versionDoc: 6
+:project_version: 7
```
Comment (stianst) [inline topics/templates/document-attributes-product.adoc]: Please send separate PR to update project versions

File: topics/templates/document-attributes-product.adoc [hunk 3]
```diff
@@ -113,14 +113,14 @@ endif::[]
 :jdgserver_version: 7.3.0
 :jdgserver_crossdcdocs_link: https://access.redhat.com/documentation/en-us/red_hat_data_grid/7.3/html/red_hat_data_grid_user_guide/x_site_replication
 
-:fuseVersion: JBoss Fuse 6.3.0 Rollup 10
+:fuseVersion: JBoss Fuse 6.3.0 Rollup 11
```
Comment (stianst) [inline topics/templates/document-attributes-product.adoc:116]: Updating Fuse versions should really be a separate PR

Comment (stianst) [reply topics/templates/document-attributes-product.adoc:116]: The above PR would need to address the broken link test for product APIs. Guess simplest is just to ignore the link.

**Review by stianst — CHANGES_REQUESTED** [review]

File: topics/templates/document-attributes-product.adoc (hunk 3 above)
Comment (hmlnarik) [reply topics/templates/document-attributes-product.adoc:116]: > Updating Fuse versions should really be a separate PR

This is the version WF 17 upgrade has been tested with, so I'd say it belongs to here.

Comment (vramik) [reply topics/templates/document-attributes-product.adoc:116]: > This is the version WF 17 upgrade has been tested with, so I'd say it belongs to here.

+1

**Review by hmlnarik — APPROVED** [review]

**Review by stianst — APPROVED** [review]
